tests/syscalls/test_bip32.c

```diff
@@ -11,59 +11,195 @@
 #include "emu_os_bip32.h"
 #include "emulate.h"
 
+#define ARRAY_SIZE(x)  (sizeof(x) / sizeof(x[0]))
+#define MAX_CHAIN_LEN  5
```

Comment:
While this is ok for BIP44, we have some projects that may require longer paths. Would it make sense to extend this to 10?

Reply:
That's just the max length used by these tests, not the syscall implementation limit ;)